src/delivery/command/init.rs

```diff
@@ -286,6 +287,22 @@ fn verify_config_get_build_cookbook_path<P>(p_path: P) -> DeliveryResult<Option<
     Ok(Some(PathBuf::from(".delivery/build_cookbook")))
 }
 
+// Create or update the delivery git remote
+pub fn create_or_update_git_remote(config: &Config) -> DeliveryResult<()> {
+    sayln("cyan", "Setting up the 'delivery' git remote...");
+    if project::verify_git_remote(config)? {
```

Comment:
I think so -- but then I'd still replace the `!=` with `==` [here](https://github.com/chef/delivery-cli/blob/0db5a932c01b552e03a62745c2e3f25b7048a480/src/delivery/project/mod.rs#L154) -- you don't want `git_remote_up_to_date` to be true if it's _not up to date_, right? 😉 

Also, there's a fair chance I'm mislead here.